nuxeo-core/nuxeo-core/src/main/resources/OSGI-INF/coldstorage-contrib.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<component name="org.nuxeo.ecm.core.event.coldstorage.contribution">
+
+  <!-- define the event that will be fired by the scheduler -->
+  <extension target="org.nuxeo.ecm.core.scheduler.SchedulerService" point="schedule">
+    <schedule id="checkAvailabilityOfBlobsBeingRetrieved">
+      <!-- every hour -->
+      <cronExpression>0 0 * ? * * *</cronExpression>
```

Comment:
You shouldn't do it at minute 0 because there's lots of schedules doing stuff at minute 0 of the hour and this causes contention. Use a random minute, like 7, instead.